src/main/scripts/janusgraph-python

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
+
```

Comment:
Add copyright header

Reply:
@sjudeng : My bad. Ideally the folder scripts shouldn't be going. This was part of my development perspective. What I've done is, I've a entry in .gitignore to ignore all files under `src/main/scripts/`